test/e2e/run.mjs

```diff
@@ -1,9 +1,11 @@
-import { dirname } from 'node:path';
-import { fileURLToPath } from 'node:url';
+/* eslint-disable no-loop-func */
```

Comment:
why is that? no related changes in the file, and it wasn't complaining before, right?

Reply:
The error that shows without adding it:

![image](https://github.com/apify/crawlee/assets/36267291/16c0a5d6-1f7b-4b97-8ea1-2b19dbc28e4c)